src/Emulator/Extensions/Utilities/GDB/Commands/ReadRegisterCommand.cs

```diff
@@ -20,12 +20,31 @@ public ReadRegisterCommand(CommandsManager manager) : base(manager)
         public PacketData Execute(
             [Argument(Encoding = ArgumentAttribute.ArgumentEncoding.HexNumber)]int registerNumber)
         {
+            var content = new StringBuilder();
+            if (registerNumber == 29)
+            {
+                //content.Append(":");
+            }
             if(!manager.Cpu.GetRegisters().Any(x => x.Index == registerNumber))
             {
+                foreach(var feature in manager.Cpu.GDBFeatures)
+                {
+                    foreach(var register in feature.Registers)
+                    {
+                        if (register.Number == registerNumber)
+                        {
+                            for(int b = 0; b < (register.Size / 4); b++)
+                            {
+                                content.AppendFormat("{0:x2}", b);
+                            }
+                            return new PacketData(content.ToString());
```

Comment:
What is the meaning of this whole section? Why should we return `00010203...` (if I understand the code correctly) for the unsupported register?

Reply:
I'm not sure if the current implementation works as expected (i.e., returning the value of 0 encoded on appropriate number of bytes).

(a) shouldn't `register.Size` be divided by 8 to get the number of bytes (I assume `register.Size` is expressed in bits, rigth?)

(b) shouldn't line 38 be
```
content.AppendFormat("{0:x2}", 0);
```

or even
```
content.Append("00")
```

Reply:
I also suggest adding a comment to the code explaining that this fragment return the default (0) value for registers defined by the architecture, but currently not implemented by the CPU model.